pylti1p3/message_launch.py

```diff
@@ -684,3 +685,8 @@ def check_observer_access(self):
         # type: () -> bool
         jwt_body = self._get_jwt_body()
         return ObserverRole(jwt_body).check()
+
+    def check_transient(self):
```

Comment:
It seems odd to not have `_access` here, but a transient role isn't really an "Access Level" as much as it is just information about the type of user launching (impersonation) that is included with roles.